apps/backend/src/resolvers/queries/QuestionsQuery.ts

```diff
@@ -22,6 +33,48 @@ export class QuestionsFilter {
   public questionIds?: string[];
 }
 
+@InputType()
+export class AllQuestionsFilter {
+  @Field(() => TemplateCategoryId, { nullable: true })
+  public category?: TemplateCategoryId;
+
+  @Field(() => [DataType], { nullable: true })
+  public dataType?: DataType[];
+
+  @Field(() => [DataType], { nullable: true })
+  public excludeDataType?: DataType[];
```

Comment:
question: It appears that the excludeDataType is not used? Shall we remove it then?